src/Race/RaceDataRepository.php

```diff
@@ -67,4 +67,31 @@ public function getActiveRaces(string $order = 'race_name', string $sort = 'ASC'
 
         return array_map(fn (array $row) => new Race($row), $data);
     }
+
+    /**
+     * @return array<array<string,int>>
```

Comment:
`array<string,int>` means it is an array with string index and int value. In this case you could either define the array with `array{name: string, cnt: int}` or return a key value pair directly from the result.

Please also note that PDO returns all value as string by default so cnt would be a string unless casted

Reply:
fixed